### PR TITLE
Resolve grammar issues throughout the site

### PR DESCRIPTION
**Pull request by jpreese**

### CODE REVIEW

**Comment by jpreese** [comment]
Good morning team! Would you prefer this to be all different pull requests? I'm seeing a lot of other merges without feedback on this one. Let me know 👍

File: pages/about.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
...
 
-Exercism provides you with thousands of exercise spread across numerous language tracks. Once you start a language track you are presented with a core set of exercises to complete. Each one is a fun interesting challenge designed to teach you a little more about the features of a language. You complete a challenge by downloading the exercise to your computer and solving it in your normal working environment. Once you've finished you submit it online and one of our mentors will give you feedback on how you could improve it using features of the language that you may not be familiar with. After a couple of rounds of refactoring, your exercise will be complete and you will unlock both the next core exercise and also a series of related side-exercises for you to practice with.
+Exercism provides you with thousands of exercises spread across numerous language tracks. Once you start a language track you are presented with a core set of exercises to complete. Each one is a fun anm interesting challenge designed to teach you a little more about the features of a language.
```
Comment (nicolechalmers) [inline pages/about.md]: Small typo here, I think "anm" should be "and".

**Comment by nicolechalmers** [comment]
This is great as one pull request, thank you!

**Comment by jpreese** [comment]
@nicolechalmers all set!

**Comment by iHiD** [comment]
Thanks so much!

**Pull request merged**